tests/mock_django_classes.py

```diff
@@ -77,6 +81,19 @@ class DjangoFeatureStateRelatedManager:
     def all(self):
         return self.feature_states
 
+    def filter(self, **kwargs):
+        # NOTE: we only implement __isnull = False
+        accepted_filters = {
```

Comment:
I don't _think_ we need any of this since we're just passing a magicmock in the test later on?

Reply:
That's the case for only one tests but tests like `test_build_identity_model_from_django_with_feature_states` depends on it.